Auth0.xcodeproj/project.pbxproj

```diff
@@ -1157,7 +1157,7 @@
 			);
 			runOnlyForDeploymentPostprocessing = 0;
 			shellPath = /bin/sh;
-			shellScript = "if which swiftlint >/dev/null; then\nswiftlint\nfi";
+			shellScript = "if which swiftlint >/dev/null && [[ `swiftlint version` =~ \"0.16\" ]] && [[ \"${CONFIGURATION}\" = \"Debug\" ]]; then\nswiftlint\nfi";
```

Comment:
If we check the configuration first there is no need to try calling the command.